src/app/contribute/guidelines/components/index.html

```diff
@@ -0,0 +1,201 @@
+    Localization
+  </stache-page-anchor>
+
+  <p>
+    To declare localization strings, specify a string name and provide the string and a description in the <stache-code>resources_en_US.json</stache-code> file in <stache-code>src/assets/locales</stache-code>. You can reference the localization strings with the <stache-code>skyLibResources</stache-code> pipe or the <stache-code>SkyLibResourcesService</stache-code> service in <stache-code>@skyux/i18n</stache-code>.
+    </p>
+
+  <stache-page-anchor>
+    Get the code
+  </stache-page-anchor>
+
+  <ol>
+    <li>
+      <a href="https://help.github.com/articles/fork-a-repo/">Fork the repo</a> into your own GitHub repo.
+    </li>
+    <li>
+      Create a branch off of master named after the feature you will contribute. The branch name should be kebab-case (my-new-feature).
+    </li>
+    <li>
+      Clone your repo locally and run <stache-code>nvm use</stache-code> to ensure you have the preferred version of Node.js installed. Next, run <stache-code>npm install</stache-code> to install all required dependencies. For more information on prerequisites, see <a stacheRouterLink="/learn/get-started/prereqs">SKY UX prerequisites</a>.
+    </li>
+    <li>
+      <a stacheRouterLink="." fragment="write-visual-regression-tests">Run your first round of visual regression tests with <stache-code>skyux e2e</stache-code></a>. This establishes baseline screenshots of the components in the master repo so that as you make changes and run your visual regression tests, your changes are validated against the state of the repo before the changes.
+    </li>
+  </ol>
+
+  <stache-page-anchor>
+    Write code
+  </stache-page-anchor>
+
+  <ol>
+    <li>
+      Launch a command prompt, <stache-code>cd</stache-code> to the folder where you cloned your branch, and run <stache-code>skyux watch --coverage library</stache-code>.
+    </li>
+    <li>
+      Launch a second command prompt, <stache-code>cd</stache-code> to the folder where you cloned your branch, and run <stache-code>skyux serve</stache-code>. This launches the SKY UX component demos page where you write the demo components to test your code.
+    </li>
+    <ul>
+      <li>
+        SKY UX demos are authored with <a href="https://github.com/blackbaud/skyux-sdk-cli">SKY UX CLI</a>. If this is your first time using SKY UX Builder, you must install the CLI as a global NPM package before you run <stache-code>skyux serve</stache-code>:
+      </li>
+      <stache-code>npm i -g @skyux-sdk/cli</stache-code>
+    </ul>
+    <li>
+      When you save code changes, the code compiles, static code analysis is performed, unit tests run, and the page refreshes with the latest changes.
+    </li>
+    <p>
+      Keep in mind that when you create components, you must add public exports through <stache-code>./src/app/public/index.ts</stache-code> to allow users to import your component modules. The public exports provide entry points for the bundles that SKY UX deploys.
```

Comment:
I see an outdated but unresolved comment from Bobby about updating this in light of the separate modules we have now. Did anyone ever follow up on this? Bobby pinged @Blackbaud-SteveBrush, @Blackbaud-AlexKingman , and @Blackbaud-TrevorBurch in his comment, but I don't see any responses.

Reply:
I replied above in-line. Let me know if you have any more questions!